terragrunt/aws/dynamodb/url_shortener_table.tf

```diff
@@ -0,0 +1,25 @@
+resource "aws_dynamodb_table" "url_shortener" {
+
+  name         = "url_shortener"
+  billing_mode = "PAY_PER_REQUEST"
+  hash_key     = "short_url"
+
+  server_side_encryption {
+    enabled = true
+  }
+
+  attribute {
+    name = "short_url"
+    type = "S"
+  }
+
+  point_in_time_recovery {
+    enabled = true
+  }
+
+  global_secondary_index {
+    name            = "ShortUrlIndex"
+    hash_key        = "short_url"
+    projection_type = "ALL"
+  }
```

Comment:
You can remove this block since we'll be able to do fast lookups using the `short_url` attribute since it's set as the `hash_key` (aka the Partition Key).

The nice thing about a global secondary index is that they can be added after the table is created if we ever discover there's another piece of table data we need to lookup a record with.

Reply:
Index has been removed - thanks for the clarification!